subt/octomap.py

```diff
@@ -176,22 +188,20 @@ def frontiers(img, start, draw=False):
     xy = np.array(xy)[:, score > limit_score]
     for dx in [-1, 0, 1]:
         for dy in [-1, 0, 1]:
-            goals.append(xy + np.repeat(np.asarray([[dy], [dx]]), xy.shape[1], axis=1))
-    goals = np.hstack(goals).T[:, ::-1]
+            for dz in [0]:  #[-1, 0, 1]:
+                goals.append(xy + np.repeat(np.asarray([[dy], [dx], [dz]]), xy.shape[1], axis=1))
+    goals = np.hstack(goals).T[:, [1, 0, 2]]
 
     # the path planner currently expects goals as tuple (x, y) and operation "in"
     goals = set(map(tuple, goals))
+    print('LEN', len(goals))
```

Comment:
Do you want this to stay in? (Just making sure.)

Reply:
OK, it was indication how bad this situation is (very large jump from 2D to 3D)